examples/DGmethods/ex_001_dycoms.jl

```diff
@@ -16,12 +16,13 @@ using LinearAlgebra
 using StaticArrays
 using Logging, Printf, Dates
 using CLIMA.VTK
-
+using DelimitedFiles
```

Comment:
Do we need all these changes to dycoms in this PR?

Reply:
Oh absolutely not, I'll have that file switched with a clean copy

Reply:
The new push should be clean